unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/NFTShape/NFTShapeFrame.cs

```diff
@@ -100,9 +106,9 @@ private void Awake()
         
         public bool IsVisible() { return gameObject.activeInHierarchy; }
 
-        public bool HasCollisions() { return boxCollider.enabled; }
+        public bool HasCollisions() { return meshCollider.enabled; }
         
-        public BoxCollider nftCollider  => boxCollider;
+        public Collider nftCollider  => meshCollider;
```

Comment:
Because since we are using a mesh collider for the PointerEvent, we need to have the same collider to it doesn't block his self when we are activating IsPointerBlocker

Reply:
not sure if I understand. why would it block itself if there should be colliders using different layers?

Reply:
I will try to explain it better.
Imagine the following situation:

1. We set the model of the NFT to `IsPointerBlocker = true`
2. The `NFTComponentHandler` will change the BoxCollider of the NFT so it blocks the pointer event
3. We add an `OnPointerDown` component to the NFT and it will create a mesh collider from the mesh to handle the event
4. Since the `BoxCollider `is bigger (because it is less accurate) than the mesh collider, it will block the pointer event raycast, so the `MeshCollider ` of the `OnPointerDown ` will never receive the `OnHover ` event since the box collider will block always the raycast

Reply:
Uhmmm, if that was a problem with the BoxCollider shouldn't we be having this same problem with ECS6 in production today? how are we dealing with that with the current ECS? I'm not sure if we are having that problem with the current ECS6 using a BoxShape, maybe we can test that with a simple test scene and see how it's currently behaving

Reply:
2- We should probably use BoxCollider for physical collision and create a MeshCollider for pointer events (including pointer blocker)
But I didn't check how does that situation is heaving currently on ECS6, cause the answer for this situation is that they should behave the same way, otherwise we are breaking compatibilities.

Reply:
We are having this problem in production today: 
![image](https://user-images.githubusercontent.com/54064092/180981053-bac6d430-09ab-45b6-9fe2-faa87703c3b2.png)

Reply:
> 2- We should probably use BoxCollider for physical collision and create a MeshCollider for pointer events (including pointer blocker) But I didn't check how does that situation is behaving currently on ECS6, cause the answer for this situation is that they should behave the same way, otherwise we are breaking compatibilities.

I don't think that this approach is a good one.  If a NFTShape comes with isPointerBlocker we will have to create 2 colliders even if it doesn't have an OnPointerDown.

We can't make ECS7 based on the compatibility with ECS6 or we will drag the problems of ECS6 into ECS7


The correct approach in my opinion is to have only 1 collider and change the layer taking into account the current status. This is no

Reply:
But NFTShape does block pointer, the reason it's not working on your example is `withCollisions = false` which looks like it's intentionally https://github.com/decentraland/unity-renderer/blob/dev/unity-renderer/Assets/Scripts/MainScripts/DCL/Components/BaseShape/BaseShape.cs#L90-L93
so it doesn't seem to have anything to do with the collider itself.
What I meant with the 2nd point it's that changing behavior it's OK only if it's extensible enough to recreate the ECS6 behavior because we can't break already deployed scenes

Reply:
Yes, but that case can always be done in the ECS6- ECS7 conversor. 

It doesn't have sense that the isPointerBlocker attribute works only if another property is true.  Nico tolds us that one time that he wanted to act just as expected, he needed to add an invisible shape to block the pointer.

I think that we should talk about this with mendez to see what we should do